de/ACMPortal.md

```diff
@@ -1,4 +1,15 @@
 ---
-redirect:   /en/ACMPortal
-layout:     redirect
+title: Fetching entries from ACM Portal
```

Comment:
Is it normal that the untranslated page is replaced by the english text? A redirect has the huge advantage that we do not need to maintain 20 english versions (or is this handled automatically by the tool?)